src/erasure-code/jerasure/jerasure_init.cc

```diff
@@ -3,7 +3,7 @@
 /*
  * Ceph distributed storage system
  *
- * Copyright (C) 2014 Cloudwatt <libre.licensing@cloudwatt.com>
```

Comment:
factoring out jerasure_init.cc and using them for both jerasure / shec should be in another commit. And this they are slightly different, a detailed explanation about why they are factored out to be the same should be provided.